pkg/pb/kt/ktarget.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package kolide.launcher;
+
+option go_package = "kt";
```

Comment:
what is `kt`? can we get a more descriptive name perhaps like `pkg/pb/targets` or something?

Reply:
renaming to `querytarget` which will remain semantically correct when we extend it beyond the simple id-serving functionality